exercises/concept/meltdown-mitigation/.meta/exemplar.py

```diff
@@ -1,14 +1,14 @@
 def is_criticality_balanced(temperature, neutrons_emitted):
     """Verify criticality is balanced.
 
-    :param temperature: int
-    :param neutrons_emitted: int
+    :param temperature: Union[int,float] temperature
```

Comment:
Please don't put type hints or items from `typing` in concept exercises.  Since they are an optional module and much further down the concept tree, they shouldn't be referred to or used at this time.

Reply:
It's inside the string, so it's not part of the code at all, no module needed. It was just one of (to me) more readable ways to say that the input could be int or float and keep it compact.

Reply:
We could also write `:param temperature: int or float value of the temperature`. Does that look better?

Reply:
Yes.  I don't want MyPy or `typing` referrals -- even in docstrings.  I just think it implies to students that they somehow need to use those things.  They don't, and it creates confusion.  It is ok to have verbose docstrings for learning.

Reply:
I appreciate wanting to be less verbose, but we also need to be really really explicit.